src/blocks/card.css

```
@@ -0,0 +1,106 @@
+.card {
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  margin-bottom: 40px;
+  height: 134px;
+  width: 160px;
+}
+
+.card__name {
+  margin-bottom: 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+
+  width: 100%;
+  margin-bottom: 16px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  width: 100%;
+  margin-bottom: 24px;
+}
+
+.card__rate-feedback {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+  height: min-content;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  width: 100%;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  text-decoration: none;
+
+  box-sizing: border-box;
+
+  width: 166px;
+  height: 40px;
+  bottom: 17px;
+  background: #00acdc;
+  border-radius: 5px;
+  border: 1px transparent solid;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.card__button:hover {
+  color: #00acdc;
+  background-color: #fff;
+
+  border: 1px solid #00acdc;
+  border-radius: 5px;
```

border radius doesn’t update on hover (without hover border radius  should be 5px too)

```suggestion
.card__button:hover {
  color: #00acdc;
  background-color: #fff;

  border: 1px solid #00acdc;
```